.gitignore

```diff
@@ -2,7 +2,6 @@
 /dist
 /node_modules
 /src/ethers
-.env
```

Comment:
Why removing the ignore for .env? Couldn't this lead to folks checking in their .env files accidentally?

Reply:
sorry, I forget to comment this. .env already ignored in line 40